tests/tests.py

```diff
@@ -135,7 +135,8 @@ def setUpTestData(cls):
         cls.concepts = [
             ResourceInstance(graph_id=SCHEMES_GRAPH_ID) for _ in range(CONCEPT_COUNT)
         ]
-        ResourceInstance.objects.bulk_create(cls.concepts)
```

Comment:
If this is what's breaking tests on `main`, could you make this into its own PR?